types.go

```diff
@@ -912,6 +914,12 @@ type Runtime struct {
 	Description      string `json:"description"`
 }
 
+type StaticIP struct {
+	Value  string
+	Family string
```

Comment:
Family is not needed if you have a net.IP, you can tell which family is it then using a golang function

Reply:
https://pkg.go.dev/net/netip#Addr.Is4

Reply:
done